### ISSUE TITLE
Under which license the tool is released ?

### ISSUE THREAD

**Issue by EmidioStani** [sparna-git/skos-testing-tool#13]
Hello @tfrancart ,

I am doing a catalogue of tools related to RDF/SKOS and I don't find the license of the skos-testing-tool.

**Comment by tfrancart** [comment]
Hello, this is LGPL 3.0. I just added a license file.
I do maintain the online service but I don't really have the bandwith to do new development on it. In general RDF data validation is now moving to SHACL-based approaches (see https://shacl-play.sparna.fr/play/).

**Comment by EmidioStani** [comment]
Thanks @tfrancart , I understand, being dependent on qskos, I can imagine also there will be no new rules to be added.

I have a question though about the rules on SHACL, do you know if the rules "SKOS Shapes (Office of Publications)" in the shapes catalogue are really maintained by OP ? 
When I tried to use them with Jena fuseki shacl tool, I had different issues and indeed I noticed another person modified them: https://github.com/skohub-io/shapes

On one hand my intention is indeed having a catalogue of tools, however for another project https://github.com/SEMICeu/Taxonomy/tree/master/Entity_Legal_Form#validation I tested with skos play!

For the rest you can close the issue.

**Comment by tfrancart** [comment]
> I have a question though about the rules on SHACL, do you know if the rules "SKOS Shapes (Office of Publications)" in the shapes catalogue are really maintained by OP ?

Yes they are published by OP at https://op.europa.eu/en/web/eu-vocabularies/application-profiles
